### ISSUE TITLE
WARNING:tensorflow:No supported ops found.

### ISSUE THREAD

**Issue by jianlong-yuan** [google-research/morph-net#23]
INFO:tensorflow:OpRegularizerManager starting analysis from: [<tf.Operation 'clone_0/semantic' type=Identity>].
INFO:tensorflow:OpRegularizerManager found 8092 ops and 0 sources.
INFO:tensorflow:OpRegularizerManager regularizing 0 groups.
WARNING:tensorflow:No supported ops found.
WARNING:tensorflow:No supported ops found.

Why no supported ops found? Is this correct?

**Comment by pkch** [comment]
This means MorphNet could not find any ops that it can use as a source to control the channel count. As a result, it created a no-op regularizer (a constant zero) -- so it's not ok.

`GammaFlopsRegularizer` looks for BatchNorm ops, `GroupLassoFlopsRegularizer` looks for Conv2D or MatMul ops. Can you double check that you have such ops in your model? If not, perhaps you use some equivalent ops that MorphNet isn't aware about -- in that case, it may be possible to extend the MorphNet regularizer logic to recognize the additional ops.

**Comment by pkch** [comment]
And thanks for mentioning this issue, we added #29 that would crash the regularizer in this case to make sure it's easily detected.
